includes/functions.php

```diff
@@ -383,6 +391,22 @@ function tsml_delete($post_ids)
 	tsml_cache_rebuild();
 }
 
+//function:	remove a meeting
+//used:	admin-import.php
+function tsml_delete_by_id($meeting_id)
```

Comment:
`tsml_delete` should be able to handle this

i think `tsml_delete_by_id(123)` is the same as `tsml_delete([123])`

Reply:
I struggled with passing it the array of ids, so I did it this way to move on. I'll revisit and get it to work...

Reply:
Done